.github/workflows/nosetests.yml

```diff
@@ -0,0 +1,41 @@
+#Github Workflow to build Debian packages for intelmq-api
+#
+#SPDX-FileCopyrightText: 2020 IntelMQ Team <intelmq-team@cert.at>
+#SPDX-License-Identifier: AGPL-3.0-or-later
+#
+name: "Run nosetest test suite"
+on:
+  push:
+    branches: [develop, maintenance, master]
+  pull_request:
+    branches: [develop, maintenance]
+
+jobs:
+  nosetest:
+    runs-on: ubuntu-latest
+    name: Run nosetest test suite
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9']
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install redis
+      run: sudo apt-get install redis -y
+
+    - name: Install test dependencies
+      run: pip install nose Cerberus pyyaml requests_mock coverage
+
+    - name: Install dependencies
+      run: pip install -e .
+
+    - name: Run tests
+      run: INTELMQ_SKIP_INTERNET=1 nosetests --with-coverage --cover-package=intelmq --cover-branches
```

Comment:
I'd prefer if we run also the tests requiring internet connectivity

Reply:
fixed